foreman/foreman.spec

```diff
@@ -190,6 +190,8 @@ BuildRequires: %{?scl_prefix_ror}rubygem(execjs) >= 1.4.0
 BuildRequires: %{?scl_prefix_ror}rubygem(execjs) < 2.5.0
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) >= 3.1
 BuildRequires: %{?scl_prefix_ror}rubygem(jquery-rails) < 4.0
+BuildRequires: %{?scl_prefix_ror}rubygem(patternfly-sass) >= 2.7
```

Comment:
Nitpick, if you can move it below jquery-ui-rails then it keeps the layout the same as the assets.rb file - making it much quicker to review changes.